checkout.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>adopt-a-dog-spring2019-checkout</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+    <body>
+      <header>
+        <h1>Seattle's premiere dog adoption agency</h1>
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="dogs.html">Dogs</a>
+          <a href="blog.html">Blog</a>
+          <a class="checkout" href="checkout.html">Checkout</a>
+        </nav>
+      </header>
+
+      <h2>Checkout</h2>
+
+    <form method="GET">
+      <label>Name
```

Comment:
You might want to rethink your label and input formatting. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label